### ISSUE TITLE
online feeding and identity supervision

### ISSUE THREAD

**Issue by Wooks-git** [NVlabs/DG-Net#78]
Hi, author. I have a question about your paper.
You used online feeding and identity supervision to get the high quality of the generated image. Is it Primary feature learning and Fine-grained feature mining loss, respectively?

**Comment by layumi** [comment]
Yes. These two losses are for the generated images x^i_j in the paper .

**Comment by Wooks-git** [comment]
Thanks for your reply. Then if don't use two losses, can't generate x^i_j images? or even if generate x^i_j images, generated low quality?

**Comment by layumi** [comment]
Hi @Wooks-git 
I do not try it, since we focus on the human representation. 

Yes. You can try it. I think the image quality will still be in a reasonable range.
